src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -392,6 +450,7 @@ const fetchRecommendedLoans = async () => {
 const loadMoreUpdates = () => {
 	updatesOffset.value += updatesLimit.value;
 	fetchUserUpdates(true);
+	updatesShown.value += UPDATES_BATCH_SIZE;
```

Comment:
We should be able to use `updatesOffset.value` instead, since it's already managing the paging, with loading 3 at a time.